src/features/entrypoint/Entrypoint.tsx

```diff
@@ -39,39 +51,74 @@ export function Entrypoint() {
     return <InstantiateContainer />;
   }
 
-  if (show === 'select-instance') {
+  if (show === 'stateless') {
+    return <RenderStateless />;
+  }
+
+  if (show === 'stateless-anon') {
+    return <RenderStateless />;
+  }
```

Comment:
These are configurable, and we should not hard-code them. The names can point to any layout-set, and this code will certainly break some applications (just not our test apps, since this here is now hard-coded to work for those).